Tools/LyTestTools/ly_test_tools/o3de/editor_test.py

```diff
@@ -149,10 +150,6 @@ class EditorTestSuite(MultiTestSuite):
     _single_test_class = EditorSingleTest
     # Test class to use for shared test collection.
     _shared_test_class = EditorSharedTest
-    # Name of the executable's log file.
-    _log_name = "editor_test.log"
-    # Executable function to call when launching Editor.
-    _executable_function = launcher_helper.create_editor
```

Comment:
I don't recall why/when this hook was added to EditorTestSuite, but it appears like it may not be an intended removal. Can you clarify why it's being removed now?

Reply:
it's intended, the reason being that when the function is placed here it binds to the test class objects. So, when `self.executable` gets set by `_executable_function` it was placing the test class as the `build` for the launcher. Moving the launcher function into the actual `_run_batched_tests()`, `_run_parallel_tests()`, `_run_parallel_batched_tests()`, and `_run_single_test()` function when setting `self.executable` got rid of this issue with the test class being placed as launch parameter.

Reply:
I parsed this as: like the "generic" launcher classes, this was added specifically to work around interacting with the MaterialEditor / AtomTools executables.